### PR TITLE
Make the customer account icon bigger to match the mini-cart one

### LINKED ISSUE #8128: Icon + text scaling on Mini Cart and My Account block
Hey folks! On the back of a chat I had with @danielwrobert, I wanted to highlight a visual inconsistency I noticed with text + icon scaling controls. 

View example, `Customer Account` and `Mini Cart` blocks both set to Typography scale `Small`: 
Screenshot 2023-01-09 at 15 32 41
Screenshot 2023-01-09 at 15 39 01

We came to the decision that for now, **we will maintain scaling the icon alongside the text using the typography settings**. However, I have noticed when the two blocks are side-by-side, the two don't look like they are using the same scale setting. 

Could we check what the settings are here and match the two? Eventually, this will also apply to the search icon scaling. 
- The mini cart icon looks slightly too big compared to the customer account icon  
- In the screenshot above, it looks like we should scale the customer account icon to visually be balanced with the mini cart icon.

Since we are keeping the settings simple, the defaults should work together well visually without having to edit the two differently. I'm including some real-world examples of where we should be aiming towards:

**01**
Screenshot 2023-01-09 at 16 08 26

**02**
Screenshot 2023-01-09 at 16 08 40

**03**
Screenshot 2023-01-09 at 16 08 55

**04**
Screenshot 2023-01-09 at 16 09 06

### PR DESCRIPTION
**Pull request by albarin**
The goal of this PR is to increase the size of the `Customer account` icon to make it more similar to the `Mini cart` one.

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/8128

#### Check the icon size
1. Go to the Site Editor and edit the Header template.
2. Insert the `Customer account` and `Mini-cart` blocks next to the menu, to look like this 👇 
Screenshot 2023-03-01 at 14 16 48

3. Check the `Customer account` icon is bigger than before and matches the `Mini cart` icon size.

| Before | After |
| ------ | ----- |
| Screenshot 2023-03-01 at 14 20 35 |Screenshot 2023-03-01 at 14 21 42 |

4. Modify the font size on both blocks to M, L, and XL and check the icon sizes also match.

#### Check the icon size of existing blocks does not change
1. Go to `trunk`.
2. Create a new page/post, add a `Customer account` block, and save.
3. Checkout to this branch (`update/8128-account-icon-size`).
4. Reload the page from step 2.
5. Insert another `Customer account` block and save.
6. Check in the editor that the block icon inserted in step 1 keeps its size and the one inserted in step 5 is bigger.
7. Check the same in the frontend.

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Fix: increase the "Customer account" size icon.

### CODE REVIEW

**Comment by albarin** [comment]
Hi @vivialice, we have increased the size of the `Customer account` icon, could you have a look to see if it's enough? Thanks!

**Comment by albarin** [comment]
@Aljullu could you have a look at this approach to increase the icon size on new blocks only?

File: assets/js/blocks/customer-account/index.tsx [hunk 1]
```diff
@@ -28,3 +29,13 @@ registerBlockType( metadata, {
...
 	},
 } );
+
+registerBlockVariation( 'woocommerce/customer-account', {
+	name: 'woocommerce/customer-account',
+	title: __( 'Customer account', 'woo-gutenberg-products-block' ),
+	isDefault: true,
+	attributes: {
+		...metadata.attributes,
+		iconClass: 'account-icon',
```
Comment (Aljullu) [inline assets/js/blocks/customer-account/index.tsx]: I would prefix this class with `wc-block-`, following [the guidelines](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/contributors/contributing/coding-guidelines.md#prefixing). This makes it more difficult that in the future another plugin or theme might break our styles. Something like: `wc-block-customer-account__account-icon`.

**Review by Aljullu — APPROVED** [review]
That's a really clever approach, I like it! I did some testing and it worked flawlessly on my end: there weren't changes in the block if it was already in the template, but when adding a new one, the sizes matched the Mini Cart block. :clap: 

I left one small suggestion below about the class name. Besides that, I would also add some explanation somewhere in the code (maybe when you register the variation) to explain the approach.

Even though the PR is a draft, I'm pre-approving, but happy to take another look if you want. :slightly_smiling_face:

**Pull request merged**